source/tutorials/learning-paths/pipelines/3-run-pipeline.rst

```diff
@@ -0,0 +1,72 @@
+.. meta::
+    :description: Valohai Pipelines learning path - Creating your first pipeline in Valohai
+
+Running a pipeline
```

Comment:
I'd change the other here:
* First show how to run this as an adhoc pipeline (and make sure you've ran vh login / vh project link etc.)
* Then if you have Git connected, show how you'd run this from the UI.

Some of our evaluating users aren't that experienced with Git, or they might not have a personal account and their company doesn't allow them to create new repos. So I'd prefer pushing for adhoc in a tutorial.